### ISSUE TITLE
Sample code in Readme.md throws exception in thrift section

### ISSUE THREAD

**Issue by sammyoos** [simplereach/helenus#80]
The following line:

``` javascript
   cf.get('foo', {consistency:Helenus.ConsistencyLevel.ONE} function(err, row){
```

from sample code in [https://github.com/simplereach/helenus#thrift]

```
throws this exception:
   node_modules/helenus/node_modules/helenus-thrift/lib/thrift/connection.js:90
      throw e;
            ^
   ReferenceError: Helenus is not defined
```

here is a simple patch that fixes it:

```
diff --git a/Readme.md b/Readme.md
index 17d6885..70ad57a 100644
--- a/Readme.md
+++ b/Readme.md
@@ -96,7 +96,7 @@ If you do not want to use CQL, you can make calls using the thrift dri

         //get what we just put in
         //the driver will return a Helenus.Row object just like CQL
-        cf.get('foo', {consistency:Helenus.ConsistencyLevel.ONE} function(err, row){
+        cf.get('foo', {consistency:helenus.ConsistencyLevel.ONE} function(err, row){
           if(err){
             throw(err);
           }
```

**Comment by calvinfo** [comment]
Good call - the change looks good to me.

**Comment by devdazed** [comment]
Fixed with commit 154790c433a7b18117d0d11664c8d07ffd32d586
